src/goCover.ts

```diff
@@ -140,6 +148,13 @@ export function getCoverage(coverProfilePath: string, showErrOutput: boolean = f
 				let fileRange = data.match(/([^:]+)\:([\d]+)\.([\d]+)\,([\d]+)\.([\d]+)\s([\d]+)\s([\d]+)/);
 				if (!fileRange) return;
 
+				// I really don't know how to fix this elsewhere. I get an underscores in front o thef path (on macOS)
+				//  in go-code-cover file. This fixes it.
+				// _/Users/me/.../file.go => /Users/me/.../file.go
+				if (fileRange[1].charAt(0) === '_') {
```

Comment:
Not quite the same, but similar. I am using gb to manage my build : https://getgb.io/ and I open the whole gb project.

I think this is not the best way to fix the problem, but understanding the source of the underscores would help. Do you know where it is coming from (and why)?